### PR TITLE
Link checker

### PR DESCRIPTION
**Pull request by adamschmideg**
Added a link checker to the travis build (and fixed what it found broken)

### CODE REVIEW

File: docs/_developers/Go-bindings-to-Ethereum-contracts.md [hunk 1]
```diff
@@ -83,7 +83,7 @@ of the contract itself, and need to specify a `backend` through which to access
 The binding generator provides out of the box an RPC backend through which you can attach
 to an existing Ethereum node via IPC, HTTP or WebSockets.
 
-We'll use the foundation's [Unicorn](https://ethereum.org/donate) token contract deployed
+We'll use the foundation's token contract deployed
```
Comment (ligi) [inline docs/_developers/Go-bindings-to-Ethereum-contracts.md]: I think it should still state unicorn token here

File: docs/_legacy/geth.md [hunk 2]
```diff
@@ -25,7 +25,7 @@ Developers and community enthusiast are advised to read the [Developers' Guide](
 ## Interfaces
 
 * Javascript Console: `geth` can be launched with an interactive console, that provides a javascript runtime environment exposing a javascript API to interact with your node. [Javascript Console API](../interface/javascript-console) includes the `web3` javascript Ðapp API as well as an additional admin API. 
-* JSON-RPC server: `geth` can be launched with a json-rpc server that exposes the [JSON-RPC API](https://github.com/ethereum/wiki/JSON-RPC)
+* JSON-RPC server: `geth` can be launched with a $c server that exposes the [JSON-RPC API](https://github.com/ethereum/wiki/wiki/JSON-RPC)
```
Comment (ligi) [inline docs/_legacy/geth.md]: $c ?

File: docs/_legacy/geth.md [hunk 3]
```diff
@@ -85,7 +85,7 @@ IRC Freenode channels:
 * `#dappdevs`: Dapp developers channel
 * `#ethdev`: buildserver etc
 
-[IRC Logs by ZeroGox](https://zerogox.com/bot/log)
+IRC Logs by ZeroGox
```
Comment (ligi) [inline docs/_legacy/geth.md]: without the link this makes no much sense anymore - IMHO should be removed completely or a replacement should be found

File: docs/_legacy/geth.md [hunk 4]
```diff
@@ -96,11 +96,11 @@ IRC Freenode channels:
 
 ### Forum
 
-- [Forum](https://forum.ethereum.org/categories/go-implementation)
+- [Forum](https://forum.ethereum.org/categories/geth)
 
 ### The ZeroGox Bot
 
-[ZeroGox Bot](https://zerogox.com/bot)
+ZeroGox Bot
```
Comment (ligi) [inline docs/_legacy/geth.md]: same as above

File: docs/_legacy/geth.md (hunk 2 above)
Comment (adamschmideg) [reply docs/_legacy/geth.md]: probably an accidental vi keybinding :)

File: docs/_developers/Go-bindings-to-Ethereum-contracts.md (hunk 1 above)
Comment (adamschmideg) [reply docs/_developers/Go-bindings-to-Ethereum-contracts.md]: done

File: docs/_legacy/geth.md (hunk 2 above)
Comment (adamschmideg) [reply docs/_legacy/geth.md]: fixed

File: docs/_legacy/geth.md (hunk 3 above)
Comment (adamschmideg) [reply docs/_legacy/geth.md]: removed

File: docs/_interface/Management-APIs.md [hunk 5]
```diff
@@ -926,6 +927,16 @@ flag.
 The `miner` API allows you to remote control the node's mining operation and set various
 mining specific settings.
 
+### miner_getHashrate
+
+TBD
```
Comment (ligi) [inline docs/_interface/Management-APIs.md]: what is this TBD about?

Comment (adamschmideg) [reply docs/_interface/Management-APIs.md]: Good catch

**Pull request merged**